Dockerfile

```diff
@@ -59,19 +58,18 @@ RUN apt-get update --assume-yes \
         zlib1g-dev \
         libsdl2-dev \
         mesa-common-dev \
-        libfreetype6-dev \
```

Comment:
Why isn't this package (or the recent equivalent) required anymore?

Reply:
It installs by dependencies.